### DISCUSSION TITLE
Repository skipped even when onboarding is set to false

### DISCUSSION THREAD

**Discussion by taraspos** [renovatebot/renovate#9066] [Request Help] [answered]
Hello,
I'm having an issue described here https://github.com/renovatebot/config-help/issues/704 (`Repository is disabled - skipping`)

Where @rarkins suggested to 
> Add a configuration manually or rename the Configure Renovate MR
 https://github.com/renovatebot/config-help/issues/704#issuecomment-644809899
 
 However, problem is that I'm using Bitbucket, where [Merged or Declined pull requests are immutable](https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D#put).

This means that the only way to enable renovate bot back is to add configuration manually, however, I don't want that, because I'm running the Renovate Bot centralized, having one config for many repos and `"onboarding": false`.

So a suggestion: maybe, Renovate Bot shouldn't skip repositories in this case when `"onboarding": false`?

Thanks!

**Comment by rarkins** [comment, marked as answer]
Why not just an empty config? `{}`

Also I'd recommend using shared presets instead of jamming everything into the bot config, unless you are the only end user (i.e. you are the only developer in every repository). Otherwise for others they'll have no visibility as to what the bot is doing or why.

**Reply by taraspos** [reply]
Yes, I had to add an empty config as a workaround, but I hoped to avoid that.

Shared config is used because we have a repository per microservice and we have like 20 of them and all of them need the same set of configurations. With the config per repository adding changes is very painful, you need to commit it into each one 🤷

**Reply by rarkins** [reply]
Many repositories all needing the same configuration is exactly what shared presets are intended for though! (don't forget to includes `extends: ["whatever"]` to your bot's `onboardingConfig` so that it's pre-populated in any onboarding PR)

**Reply by taraspos** [reply]
Thanks for the suggestion, however, I didn't see examples for configuring it with Bitbucket so wasn't sure if it is supported.
Also, if shared configuration presets are private, that means that additional permissions are needed, etc.

One more reason, why running it centralized works better for us at this point is because we are still experimenting with the best schedule to use. And since we are running renovate bot as Schedule Bitbucket Pipelines job ([similar as described here](https://chihching.net/renovate-bitbucket-cloud)), and this schedule has to be configured on each repository independently, which is one more point of pain :)
